### ISSUE TITLE
Bumped init producing odd output

### ISSUE THREAD

**Issue by langri-sha** [bumped/bumped#67]
Running `bumped init` produces the following output:

```
files:
  - package.json
plugins:
  plugins:
    prerelease:
      Linting config files:
        plugin: bumped-finepack
    postrelease:
      Generating CHANGELOG file:
        plugin: bumped-changelog
      Commiting new version:
        plugin: bumped-terminal
        command: 'git add CHANGELOG.md package.json && git commit -m "Release $newVersion"'
      Detecting problems before publish:
        plugin: bumped-terminal
        command: 'git-dirty && npm test'
      Publishing tag at Github:
        plugin: bumped-terminal
        command: 'git tag $newVersion && git push && git push --tags'
      Publishing at NPM:
        plugin: bumped-terminal
        command: npm publish
```

Looks like `{plugins: {plugins}}` is nested, so bumped isn't picking up any of the plugins.

**Comment by Kikobeats** [comment]
Thanks for report that. I fixed it in my [last commit](https://github.com/bumped/bumped/commit/beb6a5afc2f5665a3da3800b683d51bec76ec77f) but I didn't release a new version.

Now is the time!

**Comment by Kikobeats** [comment]
Published `v0.10.3`, check now please 😄

**Comment by langri-sha** [comment]
Moving from 0.10.2 to 0.10.3 and everything looks good :fireworks:
